src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java

```diff
@@ -167,7 +168,8 @@ public GoogleProjectResource initializeGoogleProject(
       GoogleProjectResource projectResource =
           resourceDao.retrieveProjectByGoogleProjectId(googleProjectId);
       UUID resourceProfileId = projectResource.getProfileId();
-      if (resourceProfileId.equals(billingProfile.getId())) {
+      if (resourceProfileId.equals(billingProfile.getId())
+          || billingProfile.getCloudPlatform() == CloudPlatform.AZURE) {
```

Comment:
When would we end up in this second condition?

Reply:
Actually at this point, we wouldn't.  I had this in there from before we merged in RBS.  At this point this should never happen.  The original thinking was for when we'd have azure datasets using the same GCP project (e.g. in dev)

Reply:
Oh I see! Thanks for clarifying